src/components/Namestage.tsx

```diff
@@ -75,43 +75,26 @@ function Namestage({
         sethasError(true);
       }
     }
-    console.log(
-      emailError === "" && toggleError === "" && usernameError === "",
-      "BUTTON"
-    );
-
-    if (emailError === "" && toggleError === "" && usernameError === "") {
-      sethasError(false);
-      setButtonstatename(true);
-    } else {
-      setButtonstatename(false);
-    }
   };
 
   const handletogglestate = (value: string) => {
     handletoggle(value);
   };
 
```

Comment:
const handletogglestate = (value: string) => handletoggle(value);
we can do like this, if its one liner function